src/Config_Command.php

```diff
@@ -421,15 +421,50 @@ public function list_( $args, $assoc_args ) {
 	 * @when before_wp_load
 	 */
 	public function get( $args, $assoc_args ) {
-		$path                = $this->get_config_path( $assoc_args );
-		$wp_config_file_name = basename( $path );
-		list( $name )        = $args;
-		$type                = Utils\get_flag_value( $assoc_args, 'type' );
-
-		$value = $this->return_value( $name, $type, self::get_wp_config_vars( $path ), $wp_config_file_name );
+		$value = $this->get_value( $assoc_args, $args );
 		WP_CLI::print_value( $value, $assoc_args );
 	}
 
+	/**
+	 * Determines whether value of a specific constant or variable defined is true.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <name>
+	 * : Name of the wp-config.php constant or variable.
+	 *
+	 * [--type=<type>]
+	 * : Type of config value to retrieve. Defaults to 'all'.
+	 * ---
+	 * default: all
+	 * options:
+	 *   - constant
+	 *   - variable
+	 *   - all
+	 * ---
```

Comment:
Should we offer a `--strict` flag too, for strict checks?

Reply:
I'm legitimately not sure what could be made stricter than it already is.

Reply:
Sorry, to clarify, `--strict` would only `WP_CLI::halt( 0 );` for `define( 'MULTISITE', true );`. It would `WP_CLI::halt( 1 );` for any truthy value (e.g. `define( 'MULTISITE', 'true' );`).

We can also leave things as they are, and wait for that to come up as a request.

Reply:
I don't know that that's particularly valuable until someone asks for it. true/false and 0/1 will already be mostly interchangeable depending on programmer's habit.